### PR TITLE
styled all components with style guidelines provided

### PR DESCRIPTION
**Pull request by mrnadaara**
Aim of this PR was to align and style components to look like a typical calculator.

Following components were styled using Sass and display flex:
- App
- Button
- ButtonPanel
- Display

### CODE REVIEW

**Review by AdnanAfsari — CHANGES_REQUESTED** [review]
**Status: Changes Required**

Hello @mrnadaara ✋ 

Everything looks perfect but just a minor fix.

- [ ] At the end of this milestone, the styled version of the calculator should look like this [Calculator](https://github.com/microverseinc/project-react-calculator/blob/master/images/calculator.png).Please make necessary changes required.

- [ ] Position the Calculator in the middle of the screen so It would look better.

Please submit for another review after making changes.

Happy Coding
Adnan Afsari

File: src/components/ButtonPanel/ButtonPanel.jsx [hunk 1]
```diff
@@ -7,32 +7,32 @@ import './ButtonPanel.scss';
...
+      <Button name="/" />
     </div>
     <div className="group">
-      <Button name="7" />
-      <Button name="8" />
-      <Button name="9" />
+      <Button name="7" color="lightgrey" />
+      <Button name="8" color="lightgrey" />
+      <Button name="9" color="lightgrey" />
       <Button name="X" />
```
Comment (bellom) [inline src/components/ButtonPanel/ButtonPanel.jsx]: I trust you can do this before merging
Kindly make this big `X` a small one `x`

File: src/components/ButtonPanel/ButtonPanel.jsx [hunk 2]
```diff
@@ -7,32 +7,32 @@ import './ButtonPanel.scss';
...
   <div className="button-panel">
     <div className="group">
-      <Button name="AC" />
-      <Button name="+/-" />
-      <Button name="%" />
-      <Button name="÷" />
+      <Button name="AC" color="lightgrey" />
+      <Button name="+/-" color="lightgrey" />
+      <Button name="%" color="lightgrey" />
+      <Button name="/" />
```
Comment (bellom) [inline src/components/ButtonPanel/ButtonPanel.jsx]: I trust you can do this 👇  before merging.
Kindly make use of this 👉  `÷` well-known division sign on a calculator.

If you have doubts, kindly check the requirement the previous reviewer shared with you.
Click on the `calculator` link

**Review by bellom — APPROVED** [review]
#### Hi @mrnadaara 

### Kindly make the little changes and move on to the next milestone 👍 

**_Good luck in the next milestone :slightly_smiling_face:_** 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**